.github/workflows/stale.yml

```
@@ -0,0 +1,30 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    # run at 12:00 every day
+    - cron: '00 12 * * *'

```

Once per week would be enough

```suggestion
    - cron: '0 0 * * SAT'

```